src/main/java/org/sagebionetworks/bridge/exporter/request/BridgeExporterRequest.java

```diff
@@ -311,44 +305,67 @@ public Builder withTag(String tag) {
-                        "absent.");
-            }
-            if (hasStartDateTime && !startDateTime.isBefore(endDateTime)) {
-                throw new IllegalStateException("startDateTime must be before endDateTime.");
+            if (startDateTime != null && !ignoreLastExportTime) {
+                throw new IllegalStateException("Cannot specify start date time without setting ignore last export date time.");
             }
 
-            // If start/endDateTime are specified, there must be a studyWhitelist.
-            if (hasStartDateTime && studyWhitelist == null) {
-                throw new IllegalStateException("If start- and endDateTime are specified, studyWhitelist must also " +
-                        "be specified.");
-            }
+            boolean hasExportType = exportType != null;
 
-            // Exactly one of date, start/endDateTime, and recordIdS3Override must be specified.
-            int numRecordSources = 0;
-            if (hasStartDateTime) {
-                numRecordSources++;
-            }
-            if (date != null) {
-                numRecordSources++;
-            }
-            if (StringUtils.isNotBlank(recordIdS3Override)) {
-                numRecordSources++;
-            }
-            if (numRecordSources != 1) {
-                throw new IllegalStateException("Exactly one of date, start/endDateTime, and recordIdS3Override must" +
-                        " be specified.");
+            // if it is a daily or hourly export, it must have endDateTime
+            boolean hasEndDateTime = endDateTime != null;
+            boolean hasRecordIdS3Override = StringUtils.isNotBlank(recordIdS3Override);
+
+            if (hasExportType) {
+                if ((exportType == ExportType.DAILY || exportType == ExportType.HOURLY) && !hasEndDateTime) {
+                    throw new IllegalStateException("Must specify endDateTime for daily or hourly export.");
+                }
+
+                if (exportType == ExportType.INSTANT && hasEndDateTime) {
+                    throw new IllegalStateException("Must not specify endDateTime for instant export.");
+                }
+
+                if (exportType == ExportType.INSTANT && studyWhitelist == null) {
+                    throw new IllegalStateException("Must specify study whitelist for instant export.");
+                }
+
+                if (exportType == ExportType.HOURLY && studyWhitelist == null) {
+                    throw new IllegalStateException("Must specify study whitelist for hourly export.");
+                }
+
+                if (hasRecordIdS3Override) {
+                    throw new IllegalStateException("Cannot specify recordIdS3Override for daily, hourly or instant export.");
+                }
+            } else {
+                if (!hasRecordIdS3Override) {
+                    throw new IllegalStateException("Must specify recordIdS3Override for override export.");
+                }
```

Comment:
Shouldn't we allow startDateTime to not have exportType? In fact, startDateTime doesn't really make sense with exportType, since we'd neither be using lastExportTime nor would we be calculating startDateTime from exportType.